.github/workflows/test.yml

```diff
@@ -105,6 +105,7 @@ jobs:
           SYSDIG_IBM_MONITOR_INSTANCE_ID: ${{ secrets.TERRAFORM_IBM_MONITOR_INSTANCE_ID }}
           SYSDIG_IBM_MONITOR_IAM_URL: "https://iam.cloud.ibm.com"
           SYSDIG_MONITOR_URL: "https://eu-gb.monitoring.cloud.ibm.com"
+          IBM_EVENT_NOTIFICATION_INSTANCE_ID: "973d1834-9731-4c39-8f04-6e860c7e4174"
```

Comment:
Should we add this to github envs instead of exposing ID like this? Is it secret?

Reply:
Even if its not secret I think we should not expose internal IDs liek this

Reply:
added a secret